### DISCUSSION TITLE
OCX components

### DISCUSSION THREAD

**Discussion by alexpamp** [XusinboyBekchanov/MyFbFramework#88] [General]
Hi I've been trying to use VFBE for a few days. Beautiful environment congratulations. I saw that it is possible to create new controls (.bi) but is it possible to use existing OCX components in a Windows environment? Thanks for the reply.

**Comment by XusinboyBekchanov** [comment]
Hello, not yet. But in the next we will add this feature too.

**Comment by alexpamp** [comment]
Prefect. Thank you
